dev/src/sample.test.html

```diff
@@ -1,432 +1,560 @@
+        </p>
+      </fieldset>
+      <p><a href="#top">[Top]</a></p>
+    </form>
+  </section>
+  <section>
+    <h2>Flex Grid</h2>
+    <div class="su-flex xs:su-gap-xs lg:su-gap-lg xl:su-gap-xl 2xl:su-gap-2xl su-text-white">
+      <div class="su-w-1/2 su-bg-sky su-rs-p-2">
+        <h3>6 of 12 columns</h3>
+        <p class="su-card-paragraph">This is a test page filled with common HTML elements to be used to provide visual feedback whilst building CSS systems and frameworks.</p>
+      </div>
+      <div class="su-w-1/2 su-bg-sky-dark su-rs-p-2">
+        <h3>6 of 12 columns</h3>
+        <p class="su-card-paragraph">This is a test page filled with common HTML elements to be used to provide visual feedback whilst building CSS systems and frameworks.</p>
+      </div>
+    </div>
+    <div class="su-flex su-grid-gap">
+      <div class="su-w-1/2 lg:su-w-3/4 su-bg-spirited su-rs-p-2">
+        <h3>8 of 12 columns</h3>
+        <p class="su-card-paragraph">This is a test page filled with common HTML elements to be used to provide visual feedback whilst building CSS systems and frameworks.</p>
+      </div>
+      <div class="su-w-1/2 lg:su-w-1/4 su-bg-spirited-dark su-rs-p-2">
+        <h3>4 of 12 columns</h3>
+        <p class="su-card-paragraph">This is a test page filled with common HTML elements to be used to provide visual feedback whilst building CSS systems and frameworks.</p>
+      </div>
+    </div>
+  </section>
+  <section>
+    <h2>Aspect Ratio Container</h2>
+    <div class="su-w-1/2">
+      <div class="su-aspect-w-16 su-aspect-h-9 su-bg-bay-dark su-text-white"><div class="su-rs-p-4 ">
+        <h3>16 x 9 box</h3>
+        <p>Note: The padding decorator classes will have to be added in a nested div because it affects the aspect ratio decorators. Similarly, to use a width decorator class, it will have to go into a parent wrapper div around the container with the aspect ratio classes.</p></div></div></div>
+  </section>
+</main>
+<footer role="contentinfo" class="su-bg-digital-red su-text-white su-text-16 md:su-text-18 su-rs-py-2">
+  <div class="su-cc">
+    <p class="su-mb-0">HTML 5 Test page by <a href="http://twitter.com/cbracco" class="su-text-white hover:su-text-white focus:su-text-white">@cbracco</a>.</p>
+  </div>
+  <a href="#top" class="su-hidden md:su-block su-fixed su-bottom-20 su-right-20 su-w-50 su-text-black hover:su-text-digital-red focus:su-text-digital-red su-transition-colors"><span class="su-sr-only">Back to top</span><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor">
+    <path fill-rule="evenodd" d="M10 18a8 8 0 100-16 8 8 0 000 16zm3.707-8.707l-3-3a1 1 0 00-1.414 0l-3 3a1 1 0 001.414 1.414L9 9.414V13a1 1 0 102 0V9.414l1.293 1.293a1 1 0 001.414-1.414z" clip-rule="evenodd" />
+  </svg></a>
+</footer>
 <!-- END DEVELOPMENT AREA -->
-</body>
+<script src="/decanter.js"></script><script src="/test.js"></script></body>
```

Comment:
you don't need to add these. Webpack should.

Reply:
I think I accidentally add them when I copy and paste from test.html to sample.test.html 😂